src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -127,9 +72,15 @@ describe('Test suite for BulkConceptsPage component', () => {
     const wrapper = mount(<Router>
       <BulkConceptsPage {...props} />
     </Router>);
-    const event = { target: { name: 'searchInput', value: 'testing' } };
+
+    // first search
+    const event = { target: { name: 'searchInput', value: '   testing' } };
     wrapper.find('#search-concept').simulate('change', event);
     wrapper.find('#submit-search-form').simulate('submit');
+    // second search
```

Comment:
@Duncanian, I think this comment is not necessary.

Reply:
Okay, let me remove it